docs/config.c

```diff
@@ -191,7 +191,7 @@
 */
 #endif
 
-{ "assumed_charset", DT_STRING, 0 },
+{ "assumed_charset", DT_SLIST|SLIST_SEP_COLON|SLIST_ALLOW_EMPTY, 0 },
```

Comment:
```suggestion
{ "assumed_charset", DT_SLIST, 0 },
```

The docs only need the base type, for now.